sycl/doc/XilinxFPGACompilation.md

```diff
@@ -31,6 +31,36 @@ Installing Xilinx FPGA compatible software stack:
   2. Xilinx runtime (XRT) for FPGAs: Download, build and install [XRT](https://github.com/Xilinx/XRT), this contains the OpenCL runtime.
   3. Xilinx SDx (2018.3+): Download and Install [SDx](https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/sdx-development-environments.html) which contains the `xocc` compiler.
 
+## Platforms
+
+It's of note that the SDx 2018.3 install comes with several platforms that do
+not work with the SYCL compiler, i.e. the ZYNC family of boards. Instead, you'll
+have to use one of the newer boards, like the Alveo U250 (*xilinx_u250_xdma_201830_1*).
+This requires some additional installation steps for the moment as it doesn't come
+packaged with the SDx download.
+
+How to:
+  1. Download the Deployment and Development Shell for your OS from the [Alveo U250 getting started page](https://www.xilinx.com/products/boards-and-kits/alveo/u250.html#gettingStarted). **Note**: you can also install the release version of the XRT runtime from here if you would rather use this than build from source.
+  2. 1. Install XRT if you haven't already: ``sudo apt install xrt_201830.2.1.1695_<OS>-xrt.deb``
```

Comment:
Say here you assume a Debian/Ubuntu Linux, there are also some RHEL/CentOS packages, but otherwise you should just install XRT from GitHub instead.